### PR TITLE
Create feature and steps files

### PR DESCRIPTION
**Pull request by reneesun**
Create feature and steps files.

### CODE REVIEW

File: features/environment.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+WAIT_SECONDS = 120
+BASE_URL = os.getenv('BASE_URL', 'http://localhost:5000')
+
+def before_all(context):
+    """ Executed once before all tests """
+    # context.driver = webdriver.PhantomJS()
+    # context.driver.manage().timeouts().pageLoadTimeout(WAIT_SECONDS, TimeUnit.SECONDS);
+    # context.driver.manage().timeouts().setScriptTimeout(WAIT_SECONDS, TimeUnit.SECONDS);
+    # context.driver.implicitly_wait(WAIT_SECONDS) # seconds
+    # context.driver.set_window_size(1120, 550)
```
Comment (iamnwi) [inline features/environment.py]: Maybe we can delete these lines as they are not used

File: features/environment.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+    options = webdriver.ChromeOptions()
+    # options.add_argument('window-size=1200x600')
```
Comment (iamnwi) [inline features/environment.py]: ditto

File: features/environment.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    options.add_argument("start-maximized") # open Browser in maximized mode
+    options.add_argument("disable-infobars") # disabling infobars
+    options.add_argument("--disable-extensions") # disabling extensions
+    options.add_argument("--disable-gpu") # applicable to windows os only
+    options.add_argument("--disable-dev-shm-usage") # overcome limited resource problems
+    options.add_argument("--no-sandbox") # Bypass OS security model
+    options.add_argument("--headless")
+    context.driver = webdriver.Chrome(options=options)
+    context.driver.implicitly_wait(WAIT_SECONDS) # seconds
+    # context.driver.set_window_size(1200, 600)
```
Comment (iamnwi) [inline features/environment.py]: ditto

File: service/service.py [hunk 4]
```diff
@@ -297,3 +297,12 @@ def initialize_logging(log_level=logging.INFO):
...
         app.logger.info('Logging handler established')
+
+######################################################################
+# DELETE ALL PET DATA (for testing only)
+######################################################################
+@app.route('/promotions/reset', methods=['DELETE'])
+def promotions_reset():
+    """ Removes all pets from the database """
+    Promotion.remove_all()
+    return make_response('', status.HTTP_204_NO_CONTENT)
```
Comment (iamnwi) [inline service/service.py]: As no test cases cover this function, so our coverage drops from 95% to 94%. Maybe better to keep this coverage at least 95%. We can either write new tests for this function or we can delete it as we didn't use it at all.

Comment (reneesun) [reply service/service.py]: Thanks for reminding me of coverage. We will use this function in steps file. I will write new tests for this.

**Pull request merged**